src/GovUk.Education.ExploreEducationStatistics.Admin/Services/EmbedBlockService.cs

```diff
@@ -73,7 +73,7 @@ public class EmbedBlockService : IEmbedBlockService
                 {
                     Id = Guid.NewGuid(),
                     Title = request.Title,
-                    Url = request.Url,
+                    Url = request.Url
```

Comment:
One for dev team to discuss I reckon?  I'm not fussed either way - it's mostly driven by Rider tutting at me!

Reply:
Yeah - I guess strictly speaking they are necessary.

I came across some other language recently (I think it may have been golang?) that enforces a final comma to the point where the code won't compile if you omit it!